EventMixing/src/ResamplingMixer_module.cc

```diff
@@ -94,6 +95,10 @@ namespace mu2e {
     spm_.beginSubRun(sr);
   }
 
+  void ResamplingMixerDetail::startEvent(art::Event const& e) {
+    spm_.startEvent(e);
+  }
+
```

Comment:
I think a similar call is needed in MixBackgroundFrames_module.  Even thought we do not plan to use the global time offset there, it will be documented as a valid setting in the module description printout as part of the helper config.  So we may as well can make it work properly.

PS. A lot of unintentional whitespace changes interermixed with real code make this file hard to review.

Reply:
I can add it.